lib/ReactViews/Map/Panels/HelpPanel/HelpPanel.jsx

```diff
@@ -71,14 +80,28 @@ class HelpPanel extends React.Component {
           css={`
             direction: ltr;
             min-width: 295px;
-            padding: 90px 20px;
+            padding: 84px 25px;
             padding-bottom: 0px;
             display: inline-block;
           `}
         >
-          <Text heading>We&apos;re here to help</Text>
-          <Spacing bottom={5} />
-          <Text medium>
+          <Text
+            css={`
+              color: #404855;
```

Comment:
Use the colors provided in theme
(either as a new prop in Text.jsx, or passing in something like
https://github.com/TerriaJS/terriajs/pull/4162/files?file-filters%5B%5D=.json&file-filters%5B%5D=.jsx&file-filters%5B%5D=.md&file-filters%5B%5D=.scss&file-filters%5B%5D=.svg&file-filters%5B%5D=.tsx#diff-c671962cbc92db588e71d07568708422R128
and
https://github.com/TerriaJS/terriajs/pull/4162/files?file-filters%5B%5D=.json&file-filters%5B%5D=.jsx&file-filters%5B%5D=.md&file-filters%5B%5D=.scss&file-filters%5B%5D=.svg&file-filters%5B%5D=.tsx#diff-2975e3acc006dfdbf49352a33d1e25f3R98

to avoid hardcoded colors), and we can raise the issue of making color palettes more consistent across each app.